src/components/scores.jsx

```diff
@@ -54,7 +54,8 @@ const Scores = ({ inst_id, play_id, single_id, send_token, isEmbedded, isPreview
 		loading: true,
 		ready: false
 	})
-	
+	const [hidePlayAgain, setHidePlayAgain] = useState(false)
```

Comment:
Is this required by the dev kit? The `hidePlayAgain` state variable was merged into the `attributes` state object, but I don't see any references to it alongside this addition.

Reply:
Nope, that might have been something that didn't get merged correctly